compiler/ras/Debug.cpp

```diff
@@ -1626,53 +1626,57 @@ TR_Debug::getName(TR::SymbolReference * symRef)
+            return "<atomicCompareAndSwapReturnValue>";
          case TR::SymbolReferenceTable::potentialOSRPointHelperSymbol:
-             return "<potentialOSRPointHelper>";
+            return "<potentialOSRPointHelper>";
          case TR::SymbolReferenceTable::osrFearPointHelperSymbol:
-             return "<osrFearPointHelper>";
+            return "<osrFearPointHelper>";
          case TR::SymbolReferenceTable::eaEscapeHelperSymbol:
-             return "<eaEscapeHelper>";
+            return "<eaEscapeHelper>";
          case TR::SymbolReferenceTable::j9VMThreadTempSlotFieldSymbol:
-             return "<j9VMThreadTempSlotFieldSymbol>";
+            return "<j9VMThreadTempSlotFieldSymbol>";
          case TR::SymbolReferenceTable::computedStaticCallSymbol:
-             return "<computedStaticCallSymbol>";
+            return "<computedStaticCallSymbol>";
          case TR::SymbolReferenceTable::j9VMThreadFloatTemp1Symbol:
-             return "<j9VMThreadFloatTemp1Symbol>";
+            return "<j9VMThreadFloatTemp1Symbol>";
          case TR::SymbolReferenceTable::objectEqualityComparisonSymbol:
-             return "<objectEqualityComparison>";
+            return "<objectEqualityComparison>";
          case TR::SymbolReferenceTable::objectInequalityComparisonSymbol:
-             return "<objectInequalityComparison>";
+            return "<objectInequalityComparison>";
          case TR::SymbolReferenceTable::nonNullableArrayNullStoreCheckSymbol:
-             return "<nonNullableArrayNullStoreCheck>";
+            return "<nonNullableArrayNullStoreCheck>";
          case TR::SymbolReferenceTable::loadFlattenableArrayElementNonHelperSymbol:
-             return "<loadFlattenableArrayElementNonHelper>";
+            return "<loadFlattenableArrayElementNonHelper>";
          case TR::SymbolReferenceTable::storeFlattenableArrayElementNonHelperSymbol:
-             return "<storeFlattenableArrayElementNonHelper>";
+            return "<storeFlattenableArrayElementNonHelper>";
          case TR::SymbolReferenceTable::J9JNIMethodIDvTableIndexFieldSymbol:
-             return "<J9JNIMethodIDvTableIndexFieldSymbol>";
+            return "<J9JNIMethodIDvTableIndexFieldSymbol>";
          case TR::SymbolReferenceTable::contiguousArrayDataAddrFieldSymbol:
-             return "<contiguousArrayDataAddrFieldSymbol>";
+            return "<contiguousArrayDataAddrFieldSymbol>";
          case TR::SymbolReferenceTable::defaultValueSymbol:
-             return "<defaultValue>";
+            return "<defaultValue>";
          case TR::SymbolReferenceTable::jitDispatchJ9MethodSymbol:
-             return "<jitDispatchJ9Method>";
+            return "<jitDispatchJ9Method>";
+         case TR::SymbolReferenceTable::jProfileValueSymbol:
+            return "<jProfileValueSymbol>";
+         case TR::SymbolReferenceTable::jProfileValueWithNullCHKSymbol:
+            return "<jProfileValueWithNullCHKSymbol>";
```

Comment:
I think the change looks fine, but I notice that most of the entries here leave out the word "Symbol" from the name that's returned.  I also notice that the entries that appear for these two symbols in [`commonNonhelperSymbolNames`](https://github.com/eclipse/omr/blob/524ce74c50c59e4f4c57c31d2e0a56769b985a9a/compiler/ras/Debug.cpp#L2154-L2155) below also include the word "Symbol".

May I ask you to remove the word Symbol from the strings for `jProfileValueSymbol` and `jProfileValueWithNullCHKSymbol`, along with those for the few other cases where Symbol appears in the String?

Reply:
Fixed it in https://github.com/eclipse/omr/pull/7503/commits/2d986c6d48d780369b2e955ed2db9dc2da8f93c6